packages/typespec-azure-resource-manager/lib/legacy/models.tsp

```diff
@@ -0,0 +1,59 @@
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+using Azure.ResourceManager.Foundations;
+using Azure.ResourceManager.Private;
+
+namespace Azure.ResourceManager.Legacy;
+
+//#region Standard Resource Operation Interfaces
+/**
+ * Concrete tracked resource types can be created by aliasing this type using a specific property type.
+ *
+ * See more details on [different Azure Resource Manager resource type here.](https://azure.github.io/typespec-azure/docs/howtos/ARM/resource-type)
+ * @template Properties A model containing the provider-specific properties for this resource
+ */
+@doc("Concrete tracked resource types can be created by aliasing this type using a specific property type.")
```

Comment:
I think these changes should be made to the standard templates, and the older ones should be marked as Legacy and deprecated.